src/OpenSpaceToolkit/Astrodynamics/Dynamics/Thruster.cpp

```diff
@@ -95,11 +95,13 @@ VectorXd Thruster::computeContribution(
         anInstant, positionCoordinates, velocityCoordinates, maximumThrustAccelerationMagnitude, aFrameSPtr
     );
 
+    const Real effectiveThrustPercent = acceleration.norm() / maximumThrustAccelerationMagnitude;
```

Comment:
is `calculateThrustAccelerationAt` not always returning the max thrust acceleration?

Reply:
So that's basically what I was wondering because:
- `ConstantThrust` Law does `const Vector3d acceleration_LOF = aThrustAcceleration * this->localOrbitalFrameDirection_.getValue();` and simply rotates the vector
- `QLaw` does `aThrustAcceleration * R_thetaRH_GCRF * thrustDirection` 
So I would expect both current laws to have effectiveThrustPercent to 1 🤔 But the change in tests seems to indicate there are currently cases where that efficiency is not 1 I guess

Useful in future cases maybe

Reply:
hum, unless this is for the current case where `QLaw` recommends not to burn but max thrust is still the default value, in which case, currently `effectiveThrustPercent` can be 1 or 0 basically (matches the test example)

Reply:
Checked, That's it I think!